cell_classification/promix_naive.py

```diff
@@ -22,7 +23,42 @@ def __init__(self, params):
         self.num_gpus = count_gpus()
         self.loss_fn = self.prep_loss()
         self.quantile = self.params["quantile"]
+        self.quantile_start = self.params["quantile"]
+        self.quantile_end = self.params["quantile_end"]
+        self.quantile_warmup_steps = self.params["quantile_warmup_steps"]
         self.class_wise_loss_quantiles = {}
+        self.aug_fn = prepare_keras_aug(params, dtype=(tf.float32, tf.uint8))
+        self.mixup_fn = MixUp(prob=params["mixup_prob"], alpha=params["mixup_alpha"])
+
+    def quantile_scheduler(self, step):
```

Comment:
Nice, I'll be curious to see how this changes the output, I imagine it will be quite helpful